integration-templates/hubspot/hubspot-service-tickets.ts

```diff
@@ -0,0 +1,97 @@
+import type { NangoSync, HubspotServiceTicket } from './models';
+
+export default async function fetchData(nango: NangoSync): Promise<{HubspotServiceTicket: HubspotServiceTicket[]}> {
+    const tickets = await paginate(nango, '/crm/v3/objects/tickets/search');
+
+    const mappedTickets: HubspotServiceTicket[] = tickets.map(ticket => ({
+        id: ticket.id,
+        createdAt: ticket.createdAt,
+        updatedAt: ticket.updatedAt,
+        archived: ticket.archived,
+        properties: {
+            content: ticket.properties.content,
+            createdate: ticket.properties.createdate,
+            hs_lastmodifieddate: ticket.properties.hs_lastmodifieddate,
+            hs_object_id: ticket.properties.hs_object_id,
+            hs_pipeline: ticket.properties.hs_pipeline,
+            hs_pipeline_stage: ticket.properties.hs_pipeline_stage,
+            hs_ticket_category: ticket.properties.hs_ticket_category,
+            hs_ticket_priority: ticket.properties.hs_ticket_priority,
+            subject: ticket.properties.subject
+        }
+    }));
+
+    if (mappedTickets.length > 0) {
+        await nango.batchSave<HubspotServiceTicket>(mappedTickets, 'HubspotServiceTicket');
```

Comment:
Batch saving all tickets at once can be memory-intensive for large accounts, and result in sync failure. 

Ideally, we should batch save after each page.